.experimenter.yaml

```diff
@@ -33,8 +33,8 @@ messaging:
       type: string
```

Comment:
chore (blocking): can we squash the commits? You can include my later suggestions and don't need to preserve authorship if it's too much trouble.